src/MemoryTracing/MemoryTracingUtils.h

```diff
@@ -5,14 +5,23 @@
 #ifndef MEMORY_TRACING_MEMORY_TRACING_UTILS_H_
 #define MEMORY_TRACING_MEMORY_TRACING_UTILS_H_
 
+#include <string_view>
+
 #include "capture.pb.h"
 
 namespace orbit_memory_tracing {
 
-std::optional<orbit_grpc_protos::SystemMemoryUsage> ParseMemInfo(
-    const std::string& meminfo_content);
+std::optional<orbit_grpc_protos::SystemMemoryUsage> ParseMemInfo(std::string_view meminfo_content);
```

Comment:
All functions that return something should be marked with `[[nodiscard]]`.

Reply:
Thanks for pointing out this! Changed accordingly.